sbhs/templates/dashboard/test_boards.html

```diff
@@ -64,5 +65,20 @@
 				</span>
 			</form>
 		</div>
-
+		{% else %}
+			<h2>
+				No boards Online
+			</h2>
+		{% endif %}
+	</div>
+</div>
+<script type="text/javascript">
```

Comment:
This can be done without the need of JavaScript!
Since the  ```slot_status``` is used to check the condition and it is static on the client.